gems/pending/MiqVm/MiqVm.rb

```diff
@@ -64,7 +70,7 @@ def vmDisks
   end
 
   def openDisks(diskFiles)
-    pVolumes = []
+    pVolumes = Array.new
```

Comment:
This must have been a change that got unchanged somehow with the rebase.  I'll change it back.